packages/file-type-icons/src/initializeFileTypeIcons.tsx

```diff
@@ -5,15 +5,35 @@ import { FileTypeIconMap } from './FileTypeIconMap';
 const PNG_SUFFIX = '_png';
 const SVG_SUFFIX = '_svg';
 
-const DEFAULT_BASE_URL = 'https://spoprod-a.akamaihd.net/files/fabric-cdn-prod_20200921.001/assets/item-types/';
-const ICON_SIZES: number[] = [16, 20, 24, 32, 40, 48, 64, 96];
+export const DEFAULT_BASE_URL = 'https://spoprod-a.akamaihd.net/files/fabric-cdn-prod_20200921.001/assets/item-types/';
+export const ICON_SIZES: number[] = [16, 20, 24, 32, 40, 48, 64, 96];
 
 export function initializeFileTypeIcons(baseUrl: string = DEFAULT_BASE_URL, options?: Partial<IIconOptions>): void {
   ICON_SIZES.forEach((size: number) => {
     _initializeIcons(baseUrl, size, options);
   });
 }
 
+// export function getFileTypeIconAsHTMLString(iconName: string, baseUrl: string = DEFAULT_BASE_URL): string | undefined {
+//   const regex = /(?<type>[a-z]+)(?<size>16|20|24|32|40|48|64|96)(_(?<pixelRatio>1.5|2|3|4)x)?_(?<suffix>png|svg)/;
+//   //
+//   const result = regex.exec(iconName);
+//   if (result && result.length) {
+//     let src = '';
+//     const type = result[1];//groups.type;
+//     const size = result[2]//groups.size;
+//     const pixelRatio = result[4];//groups.pixelRatio;
+//     const suffix = result[5];//groups.suffix;
+//     if (pixelRatio) {
+//       src = `${baseUrl}${size}_${pixelRatio}/${type}.${suffix}`
+//       return `<img src="${src}" height="100%" width="100%" />`;
+//     } else {
+//       src = `${baseUrl}${size}/${type}.${suffix}`;
+//       return `<img src="${src}" alt="" />`;
+//     }
+//   }
+// }
+
```

Comment:
Hi @dzearing @KevinTCoughlin 
Please have a look at this iteration. I had to do some refactoring and export a couple of functions from `getFileTypeIconProps` to be able to implement the function `getFileTypeIconAsHTMLString`  which is basically kind of a sister clone of  `getFileTypeIconProps` in that it accepts the same kind of object but rather than just returning the iconName, it returns the entire DOM element as a string.

The other option I had considered is this commented snippet; in which the consumer had to call `getFileTypeIconProps` to get the iconName and then just pass the iconName to the `getFileTypeIconAsHTMLString` function, but that IMO required some regEx parsing, and hence I avoided this approach.

Reply:
can you remove the commented code

Reply:
Removed.